lib/link_expansion.rb

```diff
@@ -7,9 +7,9 @@
 class LinkExpansion
   attr_reader :content_id, :locale, :with_drafts
 
-  def initialize(content_id, locale: Edition::DEFAULT_LOCALE, with_drafts: false)
+  def initialize(content_id, locale: nil, with_drafts: false)
     @content_id = content_id
-    @locale = locale
+    @locale = locale || Edition::DEFAULT_LOCALE
```

Comment:
Yeah, because we changed constructor for `ExpandedLinkSet` to accept a `nil` locale (rather than a default one of `en`), this method now gets either a `nil` locale or a set one, and if it's `nil` and we haven't given an `edition` we want it to default to `en`.